ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-TreatmentObjective-01.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Goal xmlns="http://hl7.org/fhir">
+   <id value="nl-core-TreatmentObjective-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-TreatmentObjective"/>
+   </meta>
+   <lifecycleStatus value="active"/>
+   <description>
+      <text value="Gewenste gezondheidstoestand: kan zich scheren specifiek doel: in staat om zich te scheren per 2015-02-23"/>
```

Comment:
Not really important, but maybe add a comma between the several parts for clarity (especially between the first two parts).

Reply:
Done!